projects/step-frontend/src/lib/modules/execution/components/schedule-overview/cross-execution-dashboard/cross-execution-dashboard.component.html

```diff
@@ -1,28 +1,16 @@
 <section class="header" [class.small-screen]="!!(_isSmallScreen$ | async)">
   <div class="title">
-    @if (_state.task(); as task) {
-      <h3 data-step-testid>
+    <h3 data-step-testid>
```

Comment:
I know you didn't add this now, but lets fix empty data-step-testids when touching files. I would use "task-or-plan-title"